src/state/opamFileTools.ml

```diff
@@ -396,9 +396,15 @@ let t_lint ?check_extra_files ?(check_upstream=false) ?(all=false) t =
         | #OpamUrl.version_control -> true
         | _ -> false)
   in
+  let url_archive =
+    let open OpamStd.Option.Op in
+    t.url >>| OpamFile.URL.url >>| (fun u ->
+        OpamSystem.is_archive u.OpamUrl.path)
```

Comment:
i don't think that's right. `is_archive` will look up the file locally. If you have an url like `https:///etc/fstab`, the path will be looked up.

Reply:
Good point, we should then add a function that checks if a path looks like an archive, like `looks_like_ssh_path`.

Reply:
i'm not sure we need a new function. Something like that would work just fine i think:
```suggestion
        OpamSystem.local_file u <> None)
```

Reply:
We don't want to check file existence on disk (which does `OpamUrl.local_file`). We want to know if given string seems like an archive path.

Reply:
What about using https://github.com/ocaml/opam/pull/5979 instead?